### DISCUSSION TITLE
Speed Interviewing

### DISCUSSION THREAD

**Discussion by justinnoel** [Virtual-Coffee/VC-Community-Docs#217] [Ideas]
Here was my original suggestion about the "Speed Interviewing" concept.

## VCIO Speed Interviewing
We'd have a landing page nested inside the Virtual Coffee site

`virtualcoffe.io/speed-interviewing`

### Sections on Page
* Explain the concept of the speed interviewing program
* Links
  * The current and previous cohorts of presenters
  * Signup page for candidates
    * `virtualcoffe.io/speed-interviewing/register/candidate`
  * Signup page for hiring/recruiters
    * `virtualcoffe.io/speed-interviewing/register/hiring`

## Day of Event
* Limit to 10 presenters:  ~~First come/first serve~~
  * Maybe hand pick the first batch of presenters. All folks we know have been looking for their first chance in dev jobs and we know they have the skills and confidence to present.
  * Presenters must have been members of VC for at least 3 months and attended X events
    * To prevent getting slammed by new people just trying to present
  * Presenters must self identify as “junior” or seeking first developer job
* Only registered hirers/recruiters are allowed to view on the day of the event
* Event recorded and released to public 1 week later so other possible hirers can see them.

#### Introduction
The normal VC intro with additional information about the speed interviewing event.
* 10 mins

#### Presentations
* Each Limited to 3 minutes
* Use a VC template for consistency
  * Each presenter must PR their "slide deck" using a template we create for consistency
    * Short bio, learning path, and career goals/preferences
    * Links to portfolio projects, Git repos, resume, etc
    * Testimonials, quotes, etc from references
    * ??

#### Q&A
Hirers/recruiters allowed to ask questions for 20 minutes.

Afterwards, they are free to contact the presenters if interested in discussing opportunities.

**Comment by justinnoel** [comment]
Here are the notes that @BekahHW took during our discussion:

### NEED
Many VCers are stuck in the interview grind and with some extra support, we could get them there.

1. Interest from Andrew cloudfire
2. Let’s get junior candidates more experience.
3. Tons of members with more than adequate skills that need highlighted. 

### DETAILS
Embargo the video for recruiters for 7 days. They can make a move, but if they don’t open it up to everyone.
	
### CHALLENGES OUR MEMBERS ARE FACING

- Crushed by interviews
- Crushed by rejections
- Not getting jobs even when they’re overqualified
  - Unrealistic entry-level job requirements


### WHAT WE COULD DO AS PART OF THIS

- We could work with our own companies
- Use our network
- 

### LAUNCH THIS 

- They work together to create the platform
  - landing page
  - Registration
  - Get their portfolios on there
  - Member showcase
- 5 VC members, 5 minute pitch, questions
- What if we create a typeform
  - Do you actively hire juniors?
  - Would you be interested in speed interviewing?

### RESOURCES

- Meetsy

### CONCERNS

- Vetting hiring managers
- How do we protect our members?
- Don’t want to get ppl unnecessarily excited
  - Frame it as an opportunity
  - Can we provide feedback for our members videos
- Do we act as intermediaries for members with hiring managers?
- Dry run without hiring managers is another step that they have to do.

### NEXT STEPS

- Define the plan
  - Hiring managers? recruiters?

**Comment by juliaseid** [comment]
I think this looks like a great concept.  I like the idea of doing it with a 'curated' group at first, and could certainly suggest some people who I think would do well in a first round.  These would be pre-recorded, right? So people would record, get feedback, re-record?  

At what point would we expect to know the list of hiring managers?  Where would we be recruiting them from?
